time/src/time-driver.ts

```diff
@@ -9,6 +9,8 @@ import {makeThrottleAnimation} from './throttle-animation';
 import {runVirtually} from './run-virtually';
 import {TimeSource} from './time-source';
 const requestAnimationFrame = require('raf');
+const now = require('performance-now');
```

Comment:
I am using a module called `performance-now`, it defaults to using the `performance` api if available. This does work in the browser.